lesson_16/objects/objects_app/src/test/java/com/codedifferently/lesson16/coins/CoinsTest.java

```diff
@@ -0,0 +1,91 @@
+package com.codedifferently.lesson16.coins;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.codedifferently.lesson16.coins.Coins.CoinType;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CoinTest {
+
+  @Test
+  void testCheckCoinInPocket() {
```

Comment:
The test name is unclear about what it's meant to check. It looks like you're testing the constructor, so a name like `testCoinInitialization` or `testCoinConstructor` would be a better fit.